.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[Makefile]
+indent_style = tab
+
+[*.{yml,yaml,json,js,ts}]
```

Comment:
- The two JSON files I find in Git have indention width 4 rather than two
- I find no `.js` files and no `.ts` files here

As a consequence, I would suggest to drop "js" and "ts" and "json" from this list. Thank you!

Reply:
Done.